lib/facebook/messenger/bot.rb

```diff
@@ -33,6 +33,13 @@ module Bot
         game_play
       ].freeze
 
```

Comment:
@anam-mansoor7, @taniasoaressantos 
Regarding the graph api version, I know we have some places where we use v8.0 but the developer app states v4.0: 
<img width="1605" alt="Screen Shot 2021-08-05 at 11 28 13" src="https://user-images.githubusercontent.com/15090945/128335457-62d73310-53cc-4a29-a605-d64ce87f3ad6.png">

Perhaps we could update everything to v8.0 on https://www.pivotaltracker.com/story/show/179040976?